### ISSUE TITLE
Fix tailwind issues

### ISSUE THREAD

**Issue by nathanclairmonte** [nathanclairmonte/percy-v2#3]
Having issues with tailwind and what I want to do. Seems like the dark mode features don't work well when tailwind is enabled, like only the tailwind stuff works lol. So my plan now is to go back to the main branch and then make a new branch (maybe called `test-no-tailwind` or something like that) where we will try including (i.e. copy pasting over) all the new files written in the `test-tailwind-integration` and see if they work without tailwind integrated. Shouldn't be too difficult since most of what I've written already phases out the tailwind stuff from the original repo.

**Comment by nathanclairmonte** [comment]
Finally figured out how to get dark mode working with Tailwind (wasn't working without it either lol). Decided to keep tailwind because all the youtubers I watch seem to really love it, so maybe I'll enjoy it more as I get the hang of it. 
TLDR: Fixed, tailwind stays
